### DISCUSSION TITLE
How can I create clients in command line?

### DISCUSSION THREAD

**Discussion by enrique-palazuelos** [wg-easy/wg-easy#259] [General]
Hi,

I have deployed successfully wg-easy on a kubernetes cluster. I would like to create a client automatically before (with a configmap for instance) or after the deployment in order to automate the connection from github actions and download it manually from the UI by the dev team. Is it possible?

Thanks a lot and great job! Very useful!

**Comment by faridmmv** [comment]
Agree, this feature could be useful, REST API supports downloading config, but not creating users. 
So far I found only https://github.com/linuxserver/docker-wireguard can do it.
